style.css

```diff
@@ -10,15 +8,14 @@ body {
  }
 
 h1 {
-	font-family: 'Fredoka', sans-serif;
 	color: black;
-	font-size: 25px;
+	font-size: 2vw;
```

Comment:
I am using view port width (vw) for the padding as it adjusts with any screen size